content/docs/user-guide/settings/import-settings-registry.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Settings Registry $import Chaining
+linkTitle: Settings Registry $import
+description: Learn how the Settings Registry can use the $import directive to merge additional settings registry files in Open 3D Engine (O3DE).
+weight: 300
+---
+
+The Settings Registry supports the ability to merge additional Settings Registry files \[.setreg(patch)] via using the [JSON Importer Framework](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Serialization/Json/JsonImporter.h) within the O3DE.  
+Any fields withingthe Settings Registry file that has the field name of `$import` can be used to specify additional `.setreg`, `.setregpatch` or any other json format files to merge to the Settings Registry.  
+When a Settings Registry file is merged using the [Merge API](./developer-api#merge-api), it triggers the logic to look for `$import` directives within the settings file to gather a list of additional files to merge.
+The order in which the additional files are merged are in a post-order traversal.  
```

Comment:
It might be helpful to give a brief example of post-order traversal.

Reply:
I removed the logic to have the Settings Registry files import in post-order traversal and instead, the $import files are merged as their fields are processed in top-down order.